ScoreSaber/UI/Elements/Leaderboard/ProfilePictureView.cs

```diff
@@ -83,8 +84,13 @@ internal class ProfilePictureView {
                 onFailure?.Invoke(www.error, pos);
                 yield break;
             }
-            Sprite sprite = Sprite.Create(handler.texture, new Rect(0, 0, handler.texture.width, handler.texture.height), Vector2.one * 0.5f);
-            onSuccess?.Invoke(sprite, pos, url);
+
+            // Check if cancelled properly???
+            if (!cancellationToken.IsCancellationRequested && cancellationToken != null) {
```

Comment:
This line looks like a recipe for an exception if the cancellationToken is null (wrong order of the checks)